piet-web/Cargo.toml

```diff
@@ -16,8 +16,8 @@ kurbo = "0.2.0"
 piet = { path = "../piet" }
 
 wasm-bindgen = "0.2.30"
-
+js-sys = "0.3.10"
 [dependencies.web-sys]
 version = "0.3.6"
```

Comment:
This needs to be 0.3.10, right?

Reply:
hm probably, but it seems to work without the upgrade. Strange...

Reply:
If you don’t have a `Cargo.lock` (like on CI), Cargo will pick the latest. It’s in a larger project that keeps its lock file in the repository that accurate dependencies are more useful.